collections/map/googlemap.php

```diff
@@ -347,7 +348,9 @@ function toggleLatLongDivs(){
 	<div id="map_canvas" style="width:100%;height:700px"></div>
 	<div style="width:500px;float:left;">
 		<fieldset>
-			<legend>Legend</legend>
+            <legend>
+               <?php echo (isset($LANG['LEGEND']) ? $LANG['LEGEND']: 'Legend') ?>
```

Comment:
Not sure if this will break things if not, but I've always put a semicolon at the end of the php line. Do you know if this is necessary?
` <?php echo (isset($LANG['LEGEND']) ? $LANG['LEGEND']: 'Legend')**;** ?>`

Reply:
A semi colon is only need if there are multiple lines of php code

Reply:
Good to know, thanks!